saleor/graphql/account/bulk_mutations/customer_bulk_delete.py

```diff
@@ -49,6 +50,7 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
     def bulk_action(cls, info: ResolveInfo, queryset, /):
         instances = list(queryset)
         queryset.delete()
-        manager = get_plugin_manager_promise(info.context).get()
-        for instance in instances:
-            manager.customer_deleted(instance)
+        if webhooks := get_webhooks_for_event(WebhookEventAsyncType.CUSTOMER_DELETED):
```

Comment:
Any `if` like this one directly in the Saleor code (not in plugin part) will cause, that you will not be able to use these handlers inside your own the custom plugin without declaring dummy webhook.